src/Mealz/MealBundle/Controller/SlotController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mealz\MealBundle\Controller;
+
+use App\Mealz\MealBundle\Entity\Slot;
+use App\Mealz\MealBundle\Service\SlotService;
+use Exception;
+use InvalidArgumentException;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @Security("is_granted('ROLE_KITCHEN_STAFF')")
+ */
+class SlotController extends BaseListController
+{
+    public function update(Request $request, SlotService $slotService, Slot $slot): JsonResponse
+    {
+        if ('POST' !== $request->getMethod()) {
```

Comment:
Kommen nicht-POST Anfragen überhaupt durch, da in der Routing Yaml (src/Mealz/MealBundle/Resources/config/routing.yml) ist die Methode ja auf POST gestellt? Ist der Check dann überhaupt notwendig?

Reply:
Du hast recht. Die gleiche Frage habe ich mir auch gefragt.

Ich habe es schon gefixt, aber in neuen Branch, weil es keinen Bug ist. Sehe mal in https://github.com/AOEpeople/meals/commit/0262f54d56b9180eeb622e5d4aae9a3e13b08f0c.